src/platform-integration/web/wasm.md

```diff
@@ -138,34 +168,50 @@ If the application doesn't load:
 1. Check the developer console for errors.
 1. Validate a successful build with the typical JavaScript output.
 
-[ff-preview]: https://www.mozilla.org/en-US/firefox/channel/desktop/
-
 ### Known limitations
 
 Wasm support has some limitations.
 The following list covers some common issues.
 
-#### Chrome 119 and Firefox 120 or later
-
+#### Chrome 119 or later
+ 
 As mentioned in [Load it in a browser](#load-it-in-a-browser), 
 to run Flutter web apps compiled to Wasm, 
-use _Chrome 119 or later_ or _Firefox 120 or later_.
+use _Chrome 119 or later_.
 
 Some earlier versions supported WasmGC with specific flags enabled,
 but WasmGC encodings changed once the feature was stabilized.
 To ensure compatibility, run the latest version of the Flutter `master` channel
-and the latest version of Chrome or Firefox.
+and the latest version of Chrome.
+
+- **Why not Firefox?**
+  Firefox versions 120 and later were previously able to run Flutter/Wasm, but
+  they're [currently experiencing a bug][] that is blocking compatibility with Wasm.
+- **Why not Safari?**
+  No versions of Safari have supported Wasm so far; [this bug][] tracks their
+  implementation efforts.
```

Comment:
This is where I would move the iOS WebKit note

Reply:
I moved it below the "Why not.." list but changed the wording a little, lmk if this works (nits welcome):

```
{{site.alert.warning}}
  Flutter with Wasm can't run on the iOS version of any browser.
  All browsers on iOS are forced to use the WebKit,
  and can't use their own browser engine.
{{site.alert.end}}
```

Reply:
LGTM